docs/yaml-config.rst

```diff
@@ -144,6 +144,56 @@ documentation.
 
 		conf_file: project2/docs/conf.py
 
+python.extra_requirements
+`````````````````````````
+
+* Default: ``[]``
+* Type: List
+
+List of `extra requirements`_ sections to install, additionnaly to the
+`package default dependencies`_. Only works if `python.pip_install` option
+above is set to `True`.
```

Comment:
We tend to avoid using single backtick at CPython since it means "use the default role". The more explicit way to do the same thing would be const:\`None\` but let's keep it as \`None\` for now.